.github/workflows/hungary_production.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         run: |
           cat <<EOT >> ./private/secret.ts
           export const API_KEY = '${{ secrets.IP_FIND_API_KEY }}';
+          export const SERVERLESS_URL = '${{ secrets.HU_PROD_SERVERLESS_URL }}';
```

Comment:
Just for a matter of consistency, let's change this secret name to `PROD_HU_SERVERLESS_URL`.